fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -295,6 +342,21 @@ if [ ! -e "$fMRIFolder" ] ; then
 fi
 cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
 
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction"
+    #  use FSL's fslreorient2std for reorienting the image to match the approximate orientation of the standard template images MNI152
+    #  this makes the single-band processing more robust to registration problems    
+    fslreorient2std "$fMRIFolder"/"$OrigTCSName".nii.gz "$fMRIFolder"/"$OrigTCSName"_orig_reorient.nii.gz
+    rm "$fMRIFolder"/"$OrigTCSName".nii.gz
```

Comment:
`${FSLDIR}/bin/imrm "$fMRIFolder"/"$OrigTCSName"`

Reply:
ok

Reply:
@mharms I don't actually like fsl's conventions of not having the extension on the argument, which is the only reason they had to make tools like imrm, so I would suggest easing up on this particular type of correction.

Reply:
I know you don't like that FSL convention, but given that is the predominant style that we use for working with FSL data, I find it silly to have isolated lines using 'rm', 'cp', etc, surrounded by a preponderance of lines using 'imrm', 'imcp', etc.